tests/tt_metal/tt_metal/perf_microbenchmark/noc/test_noc_unicast_vs_multicast_to_single_core_latency.cpp

```diff
@@ -11,14 +11,14 @@
 #include "tt_metal/tools/profiler/op_profiler.hpp"
 
 using namespace tt;
-
+//
 void measure_latency(string kernel_name) {
     const int device_id = 0;
     tt_metal::Device *device = tt_metal::CreateDevice(device_id);
 
-    auto dispatch_cores = device->dispatch_cores().begin();
-    CoreCoord producer_logical_core = *dispatch_cores++;
-    CoreCoord consumer_logical_core = *dispatch_cores;
+    // auto dispatch_cores = device->dispatch_cores().begin();
+    CoreCoord producer_logical_core = {0, 0};//*dispatch_cores++;
```

Comment:
why is the core here (0,0)? The yaml specifies different cores as the producer/consumer pair.